test/corpus/string.txt

```diff
@@ -1,68 +1,120 @@
-==================|||
+================================================================================
```

Comment:
Formatting changes are made automatically by `tree-sitter parse -u`

Reply:
That's unfortunate, I'll have to file a bug with tree-sitter; would you be able to add the `|||` suffix back onto everything? This is a feature I had to add to tree-sitter because the `====` and `----` test delimiter syntax was confused for TLA+ syntax.

Reply:
Reverted